oxidauth-http/src/server/api/v1/users/find_users_by_ids.rs

```diff
@@ -0,0 +1,75 @@
+use axum::{extract::State, response::IntoResponse, Json};
+use oxidauth_kernel::users::find_users_by_ids::*;
+use oxidauth_kernel::{error::IntoOxidAuthError, users::User};
+use oxidauth_permission::parse_and_validate_multiple;
+use serde::{Deserialize, Serialize};
+use tracing::{info, warn};
+use uuid::Uuid;
+
+use crate::middleware::permission_extractor::{
+    ExtractEntitlements, ExtractJwt,
+};
+use crate::provider::Provider;
+use crate::response::Response;
+
+pub type FindUsersByIdsReq = FindUsersByIds;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct FindUsersByIdsRes {
+    pub users: Vec<User>,
+    pub user_ids_not_found: Vec<Uuid>,
+}
+
+#[tracing::instrument(name = "find_users_by_ids_handler", skip(provider))]
+pub async fn handle(
+    State(provider): State<Provider>,
+    ExtractJwt(jwt): ExtractJwt,
+    ExtractEntitlements(permissions): ExtractEntitlements,
+    Json(params): Json<FindUsersByIdsReq>,
+) -> impl IntoResponse {
+    let challenges = vec!["oxidauth:users:manage".to_string()];
```

Comment:
Pretty sure we can make this `const` and move it out of the function and make it an array instead.